spring-integration-test/src/main/java/org/springframework/integration/test/context/MockIntegrationContext.java

```diff
@@ -16,10 +16,7 @@
 
 package org.springframework.integration.test.context;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.List;
+import java.util.*;
```

Comment:
Sorry, no asterisk imports.
Also, please, add your name to the `@author` list of this class.